feature-libs/quote/core/facade/quote.service.spec.ts

```diff
@@ -744,4 +757,20 @@ describe('QuoteService', () => {
       expect(savedCartFacade.editSavedCart).not.toHaveBeenCalled();
     });
   });
+
+  it('should download proposal document after calling quoteConnector.downloadAttachment', () => {
```

Comment:
MEDIUM: can we add done() here?
why? We use done() in the async context to notify the test runner that the test has reached completion.

This comment applies to all other .spec.ts files in this PR